src/main/java/org/opensearch/sdk/IndexStoreExtension.java

```diff
@@ -0,0 +1,84 @@
+package org.opensearch.sdk;
```

Comment:
Needs a license header.  Running `./gradlew spotlessApply` should add one automatically